src/formatters/deployResultFormatter.ts

```diff
@@ -8,13 +8,17 @@
 import * as chalk from 'chalk';
 import { UX } from '@salesforce/command';
 import { Logger, Messages, SfdxError } from '@salesforce/core';
-import { get, getBoolean, getString, getNumber } from '@salesforce/ts-types';
+import { get, getBoolean, getString, getNumber, asString } from '@salesforce/ts-types';
 import { DeployResult } from '@salesforce/source-deploy-retrieve';
 import {
+  CodeCoverage,
   FileResponse,
+  LocationsNotCovered,
   MetadataApiDeployStatus,
   RequestStatus,
+  Successes,
 } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import { normalizeToArray } from '@salesforce/source-deploy-retrieve/lib/src/utils';
```

Comment:
This is a bad practice.  If it's not exported at the top level then we should not rely on it.  They could rename it or move it and it would break us but that wouldn't be considered a breaking change to the library because it's not part of their public API.  Let's instead just have our own, or add it to the kit library and use it.